File: main/src/main/java/net/fwitz/math/main/binary/split/functions/exponential/SplitLogZ.java
package net.fwitz.math.main.binary.split.functions.exponential;

import net.fwitz.math.binary.split.SplitComplex;
import net.fwitz.math.plot.binary.split.SplitComplexFunctionPlot;

public class SplitLogZ {
    public static void main(String[] args) {
        new SplitComplexFunctionPlot("log(z)")
                .computeFn(SplitComplex::log)
                .render();
    }
}

File: main/src/main/kotlin/net/fwitz/math/main/binary/split/functions/exponential/SplitLogZ.kt
package net.fwitz.math.main.binary.split.functions.exponential

import net.fwitz.math.binary.split.SplitComplex
import net.fwitz.math.plot.binary.split.SplitComplexFunctionPlot

object SplitLogZ {
    @JvmStatic
    fun main(args: Array<String>) = SplitComplexFunctionPlot("log(z)")
        .computeFn(SplitComplex::log)
        .render()
}